dialogy/plugins/text/classification/xlmr.py

```diff
@@ -257,14 +303,58 @@ def train(self, training_data: pd.DataFrame) -> None:
         training_data.loc[:, const.LABELS] = encoder.transform(
             training_data[const.LABELS]
         )
-        # Add state as an additonal field to text
+
+        # Append state to text
         if self.use_state:
-            training_data[const.TEXT] += "<s> " + training_data["state"] + " </s>"
+            # logger.debug(training_data[const.STATE])
+            training_data[const.TEXT] += "<s> " + training_data[self.state_column] + " </s>"
+
```

Comment:
Please move out the addition of state / prompt to the texts in an independent function and reuse that function across `train` and `inference` methods.